packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -512,6 +1040,7 @@ it('should move thumb when pointer down on rail', () => {
   const sliderContainer = container.querySelector(
     '.iui-slider-container',
   ) as HTMLDivElement;
+
```

Comment:
It seems nitpicky, but make sure you don't make any unnecessary file changes, even if it's as small as an extra space 😄.

Reply:
**[DONE]** Ohh, right, I guess I accedentally added a line. Will remember to avoid unnecessary changes